covid_utils.py

```diff
@@ -1,3 +1,6 @@
+import datetime
```

Comment:
datetime is already imported at line 6 (import datetime as dt)

Reply:
You're right, but we may keep this for another issue ("clean up covid_utils.py") because there is a lot to do on this file and I think it's better to merge quickly: this keeps a large trunk, small branches and avoid big merge conflicts.